sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/InstancesClient.cs

```diff
@@ -445,6 +478,37 @@ public virtual Response<SearchSuggestion[]> GetSearchSuggestions(string searchSt
         /// <remarks>
         /// For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples">our repo samples</see>.
         /// </remarks>
+        /// <example>
+        /// <code snippet="Snippet:TimeSeriesInsightsSampleCreateInstance">
+        /// // Create a Time Series Instance object with the default Time Series Insights type Id.
```

Comment:
Now I am wondering if the comments need to be part of snippet.

Reply:
I believe they should be. It's better not to use this as an example, rather imagine how it would look like on the Microsoft docs. It'll give more context with comments.